<ui version="4.0" >
 <class>MediaInfoWindow</class>
 <widget class="QDialog" name="MediaInfoWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Media Info</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="mediaInfoTab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>407</width>
        <height>248</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Media Info</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4" >
       <item row="0" column="0" colspan="2" >
        <widget class="QLineEdit" name="filenameLineEdit" >
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item rowspan="2" row="1" column="0" >
        <widget class="QGroupBox" name="metadataGroupBox" >
         <property name="title" >
          <string>Metadata</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3" >
          <item row="0" column="0" >
           <layout class="QGridLayout" name="gridLayout_2" >
            <item row="0" column="0" >
             <widget class="QLabel" name="trackLabel" >
              <property name="text" >
               <string>Track #</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="titleLabel" >
              <property name="text" >
               <string>Title</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="QLabel" name="artistLabel" >
              <property name="text" >
               <string>Artist</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="0" >
             <widget class="QLabel" name="albumLabel" >
              <property name="text" >
               <string>Album</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="0" >
             <widget class="QLabel" name="yearLabel" >
              <property name="text" >
               <string>Year</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="5" column="0" >
             <widget class="QLabel" name="commentLabel" >
              <property name="text" >
               <string>Comment</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1" >
           <layout class="QGridLayout" name="gridLayout" >
            <item row="0" column="0" >
             <widget class="QLineEdit" name="trackLineEdit" >
              <property name="maximumSize" >
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="3" >
             <widget class="QLineEdit" name="titleLineEdit" />
            </item>
            <item row="2" column="0" colspan="3" >
             <widget class="QLineEdit" name="artistLineEdit" />
            </item>
            <item row="3" column="0" colspan="3" >
             <widget class="QLineEdit" name="albumLineEdit" />
            </item>
            <item row="4" column="0" >
             <widget class="QLineEdit" name="yearLineEdit" >
              <property name="maximumSize" >
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="4" column="1" >
             <widget class="QLabel" name="genreLabel" >
              <property name="text" >
               <string>Genre</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2" >
             <widget class="QLineEdit" name="genreLineEdit" />
            </item>
            <item row="5" column="0" colspan="3" >
             <widget class="QLineEdit" name="commentLineEdit" />
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QGroupBox" name="formatInfoGroupBox" >
         <property name="title" >
          <string>Format Info</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5" >
          <item row="0" column="0" >
           <widget class="QLabel" name="formatInfoLabel" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item rowspan="2" row="2" column="1" >
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>105</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" >
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="coverArtTab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>407</width>
        <height>245</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Cover Art</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
       <item row="0" column="0" >
        <widget class="QScrollArea" name="scrollArea" >
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="verticalScrollBarPolicy" >
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy" >
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable" >
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>407</width>
            <height>245</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_8" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>0</number>
           </property>
           <item row="0" column="0" >
            <widget class="QLabel" name="coverArtLabel" >
             <property name="scaledContents" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="amazonLabel" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>Cover art from &lt;a href="http://amazon.com">Amazon&lt;/a></string>
             </property>
             <property name="openExternalLinks" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="artistTab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>407</width>
        <height>245</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Artist</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9" />
     </widget>
     <widget class="QWidget" name="lyricsTab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>407</width>
        <height>245</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Lyrics</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_10" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
       <item row="0" column="0" >
        <widget class="QTextEdit" name="lyricsTextEdit" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="lyricWikiLabel" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Lyrics from &lt;a href="http://lyricwiki.org">LyricWiki&lt;/a></string>
         </property>
         <property name="openExternalLinks" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MediaInfoWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MediaInfoWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
